app/models/user.rb

```diff
@@ -1,2 +1,7 @@
 class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
+
```

Comment:
- [ ] Great job so far but I couldn't see any method return the 3 most recent for a user as required in the requirements so I recommend to add it here in user model as required 😊🔥💯🙏🏻

![image](https://user-images.githubusercontent.com/58553711/220617785-41338916-e050-4af7-857a-cad295cb4921.png)